tables/de-g2.ctb

```diff
@@ -1,25 +1,14 @@
-# liblouis: German Grade 2 Braille
-#
-# Das System der deutschen Blindenschrift (2015/2018)
-# http://www.bskdl.org/textschrift.html
-#
-# Bemerkung:
-# - Gross-/Kleinschreibung wird nicht berücksichtigt (d.h. für
-#   Abkürzungen u.ä. braucht es Vorprogramm)
-# - Akzente: Hinweis auf die beiden Tabellen (bis jetzt nur als
-#   Kommentar in de-g0.utb)
-#
-# -----------
 #-name: Deutsche Kurzschrift
 #-display-name: German contracted braille
```

Comment:
When I generate table names I use "fully contracted" if there is also a table for partially contracted braille.